walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,121 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Comparator;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Set;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
-  }
+
+    public static final int START_OF_DAY = TimeRange.START_OF_DAY;
+    public static final int END_OF_DAY = TimeRange.END_OF_DAY;
+    public static final TimeRange WHOLE_DAY = TimeRange.WHOLE_DAY;
+    public static final Comparator<TimeRange> ORDER_BY_START = TimeRange.ORDER_BY_START;
+
+
+    public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+
+        // Array of queries represenitng times to meet.
+        List<TimeRange> query = new ArrayList<>();
+        
+        // Integer represenitng the number of events.
+        int numberOfEvents = events.size();
+        
+        // Integer representing the Start of a work day.
+        int timeOfStart = START_OF_DAY;
+
+        // Integer representing the End of a work day.
```

Comment:
I appreciate how diligently you've commented your code here. I think you can afford to cut out some of the comments that only restate what the code below does. For example

```java
 // Integer representing the Start of a work day.
 int timeOfStart = START_OF_DAY;
```

The comment reads exactly the same as the code does so it doesn't provide much more insight to what's happening. Generally comments should explain *why* something is happening instead of stating what the code is doing.

Reply:
Ack.